modules/reference-guide/pages/links.adoc

```diff
@@ -0,0 +1,10 @@
+:navtitle: Links
+:keywords: reference, rule, Links
+
+= Links
```

Comment:
Side-note - none of the AsciiDoc rules are documented as yet. The reference guide has Red Hat rules only. Might be better to remove these ref guide entries for now until such time as we have an proper AsciiDoc rules section.

Reply:
Ok, I commented links.adoc and images.adoc out in the nav.adoc file so we don't have to do it again later.